samples/webview/webview.cpp

```diff
@@ -972,9 +972,11 @@ void WebFrame::OnHistory(wxCommandEvent& evt)
 void WebFrame::OnRunScript(wxCommandEvent& WXUNUSED(evt))
 {
     wxTextEntryDialog dialog(this, "Enter JavaScript to run.", wxGetTextFromUserPromptStr, "", wxOK|wxCANCEL|wxCENTRE|wxTE_MULTILINE);
-    if(dialog.ShowModal() == wxID_OK)
+    if( dialog.ShowModal() == wxID_OK )
     {
-        m_browser->RunScript(dialog.GetValue());
+        wxString result;
+        if ( m_browser->RunScript(dialog.GetValue(), &result) )
+            wxLogMessage(_("RunScript result: %s\n"), result);
```

Comment:
Okay, but I think it is going to be a lot of information because internally RunScript logs information. Done.